platform/macos/SCsub

```diff
@@ -26,7 +26,9 @@ def generate_bundle(target, source, env):
         target_bin = lipo(bin_dir + "/" + prefix, env.extra_suffix + env.module_version_string)
 
         # Assemble .app bundle and update version info.
-        app_dir = Dir("#bin/" + (prefix + env.extra_suffix + env.module_version_string).replace(".", "_") + ".app").abspath
+        app_dir = Dir(
+            "#bin/" + (prefix + env.extra_suffix + env.module_version_string).replace(".", "_") + ".app"
```

Comment:
This looks like an entirely unrelated style change

Reply:
Oh, that's strange since I didn't edit that file at all. I'm guessing pre-commit might have made some automatic edits. Should I change it back?

Reply:
Same here